src/utils/media.ts

```diff
@@ -4,8 +4,8 @@ import { config } from '../config';
 
 import { Breakpoints, DefaultTheme } from '../types/emotion';
 
-export function media(breakpoint: Breakpoints): string {
-  const breakpoints = config().grid.breakpoints;
+export function media(breakpoint: Breakpoints, theme: DefaultTheme): string {
+  const breakpoints = config(theme).grid.breakpoints;
```

Comment:
on `config` have a util `getTheme`, did you try to use it?

Reply:
I will push later. Thanks!!!!